apps/condo/domains/meter/schema/MeterReading.js

```diff
@@ -19,7 +19,11 @@ const { Meter } = require('@condo/domains/meter/utils/serverSchema')
 const { connectContactToMeterReading } = require('@condo/domains/meter/utils/serverSchema/resolveHelpers')
 const { addClientInfoToResidentMeterReading } = require('@condo/domains/meter/utils/serverSchema/resolveHelpers')
 const { addOrganizationFieldPlugin } = require('@condo/domains/organization/schema/plugins/addOrganizationFieldPlugin')
-const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { RESIDENT, SERVICE } = require('@condo/domains/user/constants/common')
+
+function canEditBillingStatusFields ({ authentication: { item: user } }) {
+    return user.isAdmin || user.isSupport || user.type === SERVICE
```

Comment:
Any service?

Reply:
Don't you need to specify, what user exactly should do?

Reply:
Only [who can edit the model](https://github.com/open-condo-software/condo/blob/e9d5784dc9c04716df8294026fe251477ee05a8a/apps/condo/domains/meter/access/MeterReading.js#L62).
Just added the test for it.